charts/argo-cd/values.yaml

```
@@ -289,14 +289,21 @@ redis:
   enabled: true
 
   image:
-    repository: redis
+    repository: quay.io/akuity/redis
     # https://hub.docker.com/_/redis/
-    tag: 7.0.8-alpine
+    tag: 7.0.4-alpine
```

I misspoke earlier. We should use:

```suggestion
    tag: 7.0.9-alpine
```